### PR TITLE
added button to delete completed tasks

### PR DESCRIPTION
**Pull request by susanthomson**

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -85,6 +86,12 @@ function reloadTodoList() {
             todoList.appendChild(listItem);
         });
         countLabel.textContent = undoneTodos;
+        if (numTodos - undoneTodos !== 0) {
+            clearCompleted.className = "someTodos";
+        }
```
Comment (jebbinSCL) [inline public/main.js]: Slight change, bracketing for the if else should be:

```
if (condition) {
   statements1
} else {
   statements2
}
```

Comment (jebbinSCL) [inline public/main.js]: For Javascript anyway

File: public/main.js [hunk 2]
```diff
@@ -117,4 +124,21 @@ function deleteTodo(id, callback) {
     };
 }
 
+function deleteCompleted() {
+    var completed = [];
+    getTodoList(function(todos) {
+        todos.forEach(function (todo) {
+            if (todo.isComplete) {
+                console.log(todo.id);
```
Comment (jebbinSCL) [inline public/main.js]: Did you mean to leave your logging statements?

File: public/main.js [hunk 3]
```diff
@@ -117,4 +124,21 @@ function deleteTodo(id, callback) {
...
+                completed.push(todo.id);
+            }
+        });
+        console.log(completed);
+        completed.forEach(function (id) {
+            deleteTodo(id, function () {});
```
Comment (jebbinSCL) [inline public/main.js]: I assume this is to be added in a later pull request? If the `function () {}` is just a place holder, its good to put a //TODO comment near it so it's clear that it's not the final code.

Comment (susanthomson) [inline public/main.js]: It's an empty callback function that deleteTodo will call. Omitting it gives "Uncaught TypeError: callback is not a function"

Comment (jebbinSCL) [inline public/main.js]: It depends: what was the reason you wanted deleteTodo to call an empty call back?

Comment (susanthomson) [inline public/main.js]: I'm potentially deleting a lot of items so if each delete is successful I don't want to do anything about it until all of the deletes are done, when the list of items will refresh.

**Comment by jebbinSCL** [comment]
:ship:

**Pull request merged**